yarn-plugin/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "yarn-plugin-spdx",
+  "private": true,
+  "main": "./sources/index.ts",
+  "dependencies": {
+    "@yarnpkg/cli": "^4.0.0",
+    "@yarnpkg/core": "^4.0.0",
+    "@yarnpkg/fslib": "^3.0.0",
+    "@yarnpkg/libzip": "^3.0.0",
+    "@yarnpkg/parsers": "^3.0.0",
+    "@yarnpkg/plugin-pnp": "^4.0.0",
+    "clipanion": "^4.0.0-rc.2"
+  },
+  "devDependencies": {
+    "@types/node": "^18.0.0",
+    "@yarnpkg/builder": "^4.0.0",
+    "rimraf": "5.0.0",
+    "typescript": "5.3.0-beta"
```

Comment:
a bit unusual to use a beta version here, especially since the root package.json uses another version of typescript.

Reply:
right, I am not sure how this one came in. Changed it to the same version that is used in the root.